Libplanet/Net/Transports/NetMQTransport.cs

```diff
@@ -518,6 +524,7 @@ public void ReplyMessage(Message message)
 
         private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
         {
+            _logger.Verbose("ReceiveMessage started. {@E}", e);
```

Comment:
How about using `nameof` keyword?

```suggestion
            _logger.Verbose($"{nameof(ReceiveMessage)} started. {{@E}}", e);
```

Reply:
Thanks for the comment, but sadly this line was used in local testing and added to vcs by my mistake 😢 . I'll remove this.